mer_verify_kernel_config

```diff
@@ -151,7 +151,8 @@ CONFIG_CGROUP_PERF		y,!	# systemd (optional): http://0pointer.de/blog/projects/c
 CONFIG_CGROUP_SCHED		y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html
 CONFIG_BLK_CGROUP		y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html
 CONFIG_NET_CLS_CGROUP		y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html
-CONFIG_NETPRIO_CGROUP		y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html
+CONFIG_NETPRIO_CGROUP		y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html, ignore if kernel version is >=3.15
+CONFIG_CGROUP_NET_PRIO          y,!     # see above, ignore if kernel version is <3.15
```

Comment:
Please write the whole explanation, because the comment (after #) will be visible when this script is run, and they won't know what to find "above"

Reply:
done.